src/base/bittorrent/session.h

```diff
@@ -330,6 +334,16 @@ namespace BitTorrent
         void setDiskCacheTTL(uint ttl);
         bool useOSCache() const;
         void setUseOSCache(bool use);
+        bool guidedReadCache() const;
```

Comment:
@glassez:  
Should `useOSCache` and other preexisting `bool` names be changed as well?